go/tools/asthelpergen/asthelpergen.go

```diff
@@ -92,54 +93,41 @@ func newGenerator(mod *packages.Module, sizes types.Sizes, named *types.Named, g
 }
 
 func findImplementations(scope *types.Scope, iff *types.Interface, impl func(types.Type) error) error {
+	const OnlyReferences = false
```

Comment:
this can be lowercase?

Reply:
Constants in Go are always UpperCase or the linter will complain!

Reply:
@vmg Huh? The first letter is about exporting it or not? You can for sure have constants that start with a lowercase?